README.md

```diff
@@ -54,7 +54,12 @@ File `package.json`:
     "scriptPreprocessor": "<rootDir>/__tests__/preprocessor.js",
   },
   "jest-webpack-alias": {
-    "webpackProfile": "dev"
+    // Required. If non-null value provided, will expect to find your webpack
+    // config stored at this key. Set to null for a regular webpack config file.
```

Comment:
Ah, interesting! I was ignorant of this. Still, though, this doesn't seem to match up with the implementation here. In that example, an array of objects is exported. But `jest-webpack-alias` tries to look up a key as if it was expecting an object of objects. So is this multi-compiler stuff the same thing?